lib/CppInterOp/CppInterOp.cpp

```diff
@@ -3422,6 +3448,8 @@ static Decl* InstantiateTemplate(TemplateDecl* TemplateD,
   // This will instantiate tape<T> type and return it.
   SourceLocation noLoc;
   QualType TT = S.CheckTemplateIdType(TemplateName(TemplateD), noLoc, TLI);
+  if (TT.isNull())
+    return nullptr;
```

Comment:
I'd drop that if we can't cover it with tests.

Reply:
This is required; it is due to a template instantiation failing in Eigen (crashes).
Also, fixing the instantiation in this PR does not make sense. Will look into that once I get to the eigen tests.
And writing a test that is supposed to fail is bad again.

Reply:
Ok. Understood

Reply:
Let’s merge it then.